.github/workflows/lint-pr.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   main:
-    runs-on: ubuntu-latest
```

Comment:
can we keep come that don't need to be sped up on the normal one from git?

Reply:
yep!